modules/mono/editor/bindings_generator.cpp

```diff
@@ -150,8 +150,268 @@ static String fix_doc_description(const String &p_bbcode) {
 			.strip_edges();
 }
 
+String BindingsGenerator::bbcode_to_text(const String &p_bbcode, const TypeInterface *p_itype) {
```

Comment:
Don't we have a method in RichTextLabel for that already? You can make a RTL and have it parse the bbcode to retrieve the text only version.

Reply:
Ah I see it's doing C# specific replacements, so I guess it does need an ad hoc reimplementation.

Reply:
Well, I need to convert the referenced types, methods, and members to the C# names (e.g.: `my_method` -> `MyMethod`) and I don't think a method in RichTextLabel would do that for me.